### PR TITLE
Fixed league container support

### PR DESCRIPTION
**Pull request by paolooo**
See https://github.com/bernardphp/bernard/issues/292

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,11 +21,11 @@
         "doctrine/dbal": "~2.3",
         "aws/aws-sdk-php": "~2.4|~3.0",
         "pda/pheanstalk": "~3.0",
-        "league/container": "~1.0",
         "php-amqplib/php-amqplib": "~2.5",
         "phpspec/phpspec": "^2.4",
         "phpunit/phpunit": "^4.8",
-        "iron-io/iron_mq": "~4.0"
+        "iron-io/iron_mq": "~4.0",
+        "league/container": "~2.3.0"
```
Comment (henrikbjorn) [inline composer.json]: should be `~2.3`

Comment (paolooo) [inline composer.json]: ok

Comment (paolooo) [inline composer.json]: done @henrikbjorn

Comment (paolooo) [inline composer.json]: Please see https://github.com/bernardphp/bernard/pull/293#pullrequestreview-22962659

**Comment by paolooo** [comment]
@henrikbjorn - please review my changes. tnx

**Pull request merged**